File: src/main/java/entities/PointDistribution.java
package entities;

public class PointDistribution {
    private  int idPointDistribution;
    private  String nom;
    private  String adresse;
    private  int idLivraison;

    public PointDistribution() {
        this(0, null, null, 0);
    }

    public PointDistribution(int idPointDistribution, String nom, String adresse, int idLivraison) {
        this.idPointDistribution = idPointDistribution;
        this.nom = nom;
        this.adresse = adresse;
        this.idLivraison =idLivraison;
    }

    public PointDistribution(String nom, String adresse, int idLivraison) {
        this.nom = nom;
        this.adresse = adresse;
        this.idLivraison = idLivraison;
    }

    public int getIdPointDistribution() {
        return idPointDistribution;
    }

    public void setIdPointDistribution(int idPointDistribution) {
        this.idPointDistribution = idPointDistribution;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public int getIdLivraison() {
        return idLivraison;
    }

    public void setIdLivraison(int idLivraison) {
        this.idLivraison = idLivraison;
    }

    @Override
    public String toString() {
        return "PointDistribution{" +
                "idPointDistribution=" + idPointDistribution +
                ", nom='" + nom + '\'' +
                ", adresse='" + adresse + '\'' +
                ", idLivraison=" + idLivraison +
                '}';
    }
}

File: src/Entity/PointDistribution.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Table(name: "point_distribution")]
#[ORM\Entity(repositoryClass: "App\Repository\PointDistributionRepository")]
class PointDistribution
{
    #[ORM\Id]
    #[ORM\GeneratedValue(strategy: "IDENTITY")]
    #[ORM\Column(name: "id_point_distribution", type: "integer", nullable: false)]
    private int $idPointDistribution;

    #[ORM\Column(name: "nom", type: "string", length: 255, nullable: false)]
    private string $nom;

    #[ORM\Column(name: "adresse", type: "string", length: 255, nullable: false)]
    private string $adresse;

    #[ORM\Column(name: "id_livraison", type: "integer", nullable: false)]
    private int $idLivraison;

    public function getIdPointDistribution(): ?int
    {
        return $this->idPointDistribution;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): static
    {
        $this->nom = $nom;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): static
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getIdLivraison(): ?int
    {
        return $this->idLivraison;
    }

    public function setIdLivraison(int $idLivraison): static
    {
        $this->idLivraison = $idLivraison;

        return $this;
    }
}
